### PR TITLE
docs: add tensorboard user guide

### PR DESCRIPTION
**Pull request by brnovasco**
Requested user guide for tensorboard.

### CODE REVIEW

**Comment by brnovasco** [comment]
This page would be better placed at the controls and pages sections. This also needs an introduction and fixing other pages that mention this feature.

File: apps/docs/content/docs/concepts/index.mdx [hunk 1]
```diff
@@ -36,4 +36,10 @@ impacts the results and learning analysis.
     description="Results and analysis."
     href="/docs/concepts/inference"
   />
+  <Card
+    title="Tensorboard"
+    description="Visualistion."
```
Comment (matyson) [inline apps/docs/content/docs/concepts/index.mdx]: typo

**Review by matyson — CHANGES_REQUESTED** [review]

File: apps/docs/content/docs/concepts/index.mdx (hunk 1 above)
Comment (brnovasco) [reply apps/docs/content/docs/concepts/index.mdx]: This card should not be there.

**Review by matyson — APPROVED** [review]
looks great, thanks

**Pull request merged**